src/router/handlers/test.js

```diff
@@ -0,0 +1,10 @@
+const dal = require('../../dal/dal');
+
+let test = async (req, res) => {
+    dal.testFunction('dal test').then((res) => console.log(res));
```

Comment:
What is "dal"? Right now, my brain is thinking "Dalhousie University" and that's it 😂

Reply:
Database Access Layer! Haha. Your head's in Halifax - _literally_.